Xamarin.Forms.Platform.Android/IButtonLayoutRenderer.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System;
 using Android.Support.V7.Widget;
+using AButton = Android.Widget.Button;
 
 namespace Xamarin.Forms.Platform.Android
 {
 	public interface IButtonLayoutRenderer
 	{
-		AppCompatButton View { get; }		
+		AButton View { get; }		
```

Comment:
This is a breaking change. Why are we changing the type?

Reply:
Yeah @PureWeen made a comment about that too in the original PR with a proposed solution I think. So, I'll look at that

Reply:
Implemented proposed solution